MetaMask/metamask-extension
# Added metric Phishing Page Displayed for phishing warning page

## Explanation
Added metric Phishing Page Displayed for phishing warning page, that occurs whenever user triggers phishing warning page.

* Fixes #17453

## Screenshots/Screencaps
<img width="719" alt="Screenshot 2023-01-27 at 13 25 33" src="https://user-images.githubusercontent.com/120601632/215086826-bcba631e-1908-4b41-9933-636ca94022cb.png">

## Manual Testing Steps

1. Build project and run mock segment
2. Login to extension
3. Go to one of the untrusted sites: primalfrens.club, geishateahouse.io, cryptocity.fun
4. Phising page is opened
5. Check in console where the mock segment is, Phising Page Displayed metric is shown
